### PR TITLE
Use same copy as elsewhere

### PR DESCRIPTION
**Pull request by joelochlann**
This control copy in #20742 was supposed to be the same as elsewhere - I mistakenly used the custom copy for the US designed for the `In case you missed it ...` hook, but setting the hook to the same as elsewhere.

## before - control
old-control

## after - control
new-control

@ionamckendrick

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-pre-end-of-year.js [hunk 1]
```diff
@@ -41,15 +41,14 @@ export const acquisitionsEpicUsPreEndOfYear: EpicABTest = makeABTest({
...
-                        'If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.',
+                        '… three years ago, we knew we had to try to make The Guardian sustainable by deepening our relationship with our readers. The revenues from our newspaper had diminished and the technologies that connected us with a global audience had moved advertising money away from news organisations. We knew we needed to find a way to keep our journalism open and accessible to everyone, regardless of where they live or what they can afford.',
```
Comment (jranks123) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-pre-end-of-year.js]: `&hellip;`

**Review by jranks123 — APPROVED** [review]
with comment

**Comment by ionamckendrick** [comment]
Looks good!

**Review by ionamckendrick — COMMENTED** [review]
Great!

**Pull request merged**